### ISSUE TITLE
animal ally dose not work like it dose on table top

### ISSUE THREAD

**Issue by Gisellle** [Vek17/TabletopTweaks-Base#745]
when you have animal ally its is supost to stack with other sources of animal companions acquired after getting the feat 
for example my when 5th level slayer picks up animal ally she gets a lv2 animal companion then when she levels up and picks up a level in hunter her animal companion on the table top would go to lv4 one from the character level one from the hunter level but it only gets the one from the character level in your current implementation 
here is animal ally for context 
Animal Ally

Your respect for nature is so great that you can form a deep and lasting friendship with an animal.

Prerequisites: Nature Soul, character level 4th, must not have an animal companion or mount that advances as an animal companion.

Benefit: You gain an animal companion as if you were a druid of your character level –3 from the following list: badger, bird, camel, cat (small), dire rat, dog, horse, pony, snake (viper), or wolf. If you later gain an animal companion through another source (such as the Animal domain, divine bond, hunter’s bond, mount, or nature bond class features), **the effective druid level granted by this feat stacks with that granted by other sources.**

i hope you can get this fixed soon i'm just going to add boon companion once i hit hunter 3 to see if it will work as a temp fix in the mean time thank you for taking the time to read this

**Comment by Vek17** [comment]
These would not stack at all in tabletop because that is not how animal
companions work in tabletop. Advancing existing companions is an owl at me
panic to avoid multiple companions.

**Comment by Gisellle** [comment]
did you not see the text of the feat 
 "**the effective druid level granted by this feat stacks with that granted by other sources.**"

**Comment by Vek17** [comment]
The test refers to effects like boon companion. It does not let you get two
levels worth of druid level every level up.

**Comment by Gisellle** [comment]
"**the effective druid level granted by this feat stacks with that granted by other sources.**" 

how dose this explicitly exclude the class features and only include boon companion? witch btw one of the only feats that buffs like that the only other thing i can think of that raises animal companion effective druid level is horse master and that dose it by character level thus getting you 2 per level 

unless you are saying by getting a gaining an animal companion through another source i am breaking the Prerequisites and should not be benefiting from it. but then i have to ask why is that text "If you later gain an animal companion through another source" there?

i will say it should have a cap like boon companion where it can not exceed your character level 

but by the wording of it it should work like a hunter with a cav they use almost the same wording for level stacking and it has a explicit line of text for if you break the prerequisite so i don't see how it would work out

**Comment by Gisellle** [comment]
if we enforce the can not exceed your character level cap like boon companion so you can't rush a lv20 animal companion like raw 
it dose not seem like it would break the game ether you can get a lv20 animal companion on a class that dose not get them normaly in 3 other ways already 

go 4 lvs in cav to get expert trainer for the horse master feat then the rest what ever 
pick up nature domain be it a 1 level dip or your main class and get boon companion
or grab nature soul, animal ally, and boon companion

so having this dose not seem out of place 

go 5 lvs in a class grabbing nature soul and animal ally feats then 3lvs in a class that has a animal companion

**Comment by Gisellle** [comment]
hold on the feat self detonates by raw 

as if you don't meet the Prerequisites for a feat you don't get to benefit from it anymore 

so it would turn its self on and off over and over again as it needs you to "not have an animal companion or mount that advances as an animal companion." but it gives you one

**Comment by Vek17** [comment]
The more important factor is other classes do not advance animal companions
in tabletop. You would gain an additional animal companion starting at
level 1 scaled to that class. This whole interaction cannot exist in
tabletop so I do not even consider it. It is simply the result of an owlcat
change to the game in thier implementation.

**Comment by Gisellle** [comment]
so i get that that a thing if you PICK DIFFERENT ANIMALS but if you have the same they stack

https://paizo.com/paizo/faq/v5748nruor1fn#v5748eaic9qqn

"If the animal is on the cavalier mount list and on the list of animal companions for your other class, your cavalier and druid levels stack to determine the animal's abilities. If the animal is not on the cavalier mount list, the druid levels do not stack and you must have different animals (one an animal companion, one a cavalier mount).
For example, if you are Medium druid and you choose a horse companion, levels in cavalier stack to determine the horse's abilities. If you are a Medium druid and you choose a bird companion, levels in cavalier do not stack to determine the bird's abilities, and you must choose a second creature to be your mount (or abandon the bird and select an animal companion you can use as a mount).
This same answer applies to multiclassed cavalier/rangers.
(Note that the design team discourages players from having more than one companion creature at a time, as those creatures tend to be much weaker than a single creature affected by these stacking rules, and add to the bookkeeping for playing that character.)"

in the case of my pc its wolf from Animal Ally and wolf from hunter so they stack

**Comment by Vek17** [comment]
I am not sure what you describe is intentional, but even if it is after looking at what would be required to create that behavior I am just going to close this as "won't fix". I think the current implementation covers practical cases and the amount of work required to get your desired behavior is too large given I am not confident that your "druid level" being higher than your actual level is an intended way for any of this to work.
